### PR TITLE
chore(coverage.codecov): publish coverage report to codecov

### PR DESCRIPTION
**Pull request by corinnekrych**
ready to merge

### CODE REVIEW

**Comment by corinnekrych** [comment]
@kwk @joshuawilson I don't have admin right on the repo. we need to activate codecov via settings tab and get the token which should be replaced in: 
https://github.com/fabric8io/fabric8-ui/pull/1368/files#diff-b1bda5af9b032c2f6d3a85e1bb6b2f2eR15 

@kwk is it all we need to get the nice graph with cumulative coverage report like: https://codecov.io/gh/fabric8io/fabric8-planner ?

**Comment by fabric8cd** [comment]
@corinnekrych snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1368-fabric8-ui.int.rdu2c.fabric8.io

**Comment by joshuawilson** [comment]
Should we really be placing a token in github? That would grant anyone admin rights to the whole org, right?

**Comment by kwk** [comment]
> Should we really be placing a token in github? That would grant anyone admin rights to the whole org, right?

@joshuawilson we sure shouldn't but I think @maxandersen 's [secret storage PR](https://github.com/almighty/almighty-jobs/pull/17) has not made it in to the almighty-jobs repo yet. I guess that is a potential location for tokens.

**Comment by kwk** [comment]
@corinnekrych I've just discovered this new feature in codecov.io. They support categorizing different tests into whatever categories you want: https://docs.codecov.io/docs/flags . I'd highly suggest to take a look at it.

**Comment by maxandersen** [comment]
> Should we really be placing a token in github? That would grant anyone admin rights to the whole org, right?
> @joshuawilson we sure shouldn't but I think @maxandersen 's secret storage PR has not made it in to the almighty-jobs repo yet. I guess that is a potential location for tokens.

Two things: 
My PR was mainly just for storing the credentials - it is just yaml and the build system need to somehow grok it. If we move to fabric8 ci that should be looked at.

you should already now go and change the token as it is now public - and I believe git hub will eventually disable the access as they scan git pushes for credentials.

**Comment by fabric8cd** [comment]
@corinnekrych snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1368-fabric8-ui.int.rdu2c.fabric8.io

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 19823405c6ba7908ca8ef0eada05fc160ff78d04
  * Your subject line is longer than 50 characters

 * Commit: 2073a9093297a7def14228a2ea899cb7b09ea97d
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by fabric8cd** [comment]
@corinnekrych snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1368-fabric8-ui.int.rdu2c.fabric8.io

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 19823405c6ba7908ca8ef0eada05fc160ff78d04
  * Your subject line is longer than 50 characters

 * Commit: 2073a9093297a7def14228a2ea899cb7b09ea97d
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 2b3b0baf3cfbd4c6f51f8a883eb35a4288ed9c09
  * Your subject line is longer than 50 characters

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 19823405c6ba7908ca8ef0eada05fc160ff78d04
  * Your subject line is longer than 50 characters

 * Commit: 2073a9093297a7def14228a2ea899cb7b09ea97d
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 2b3b0baf3cfbd4c6f51f8a883eb35a4288ed9c09
  * Your subject line is longer than 50 characters

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by fabric8cd** [comment]
@corinnekrych snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1368-fabric8-ui.int.rdu2c.fabric8.io

**Comment by fabric8cd** [comment]
@corinnekrych snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1368-fabric8-ui.int.rdu2c.fabric8.io

**Comment by joshuawilson** [comment]
[test]

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 19823405c6ba7908ca8ef0eada05fc160ff78d04
  * Your subject line is longer than 50 characters

 * Commit: 2073a9093297a7def14228a2ea899cb7b09ea97d
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 2b3b0baf3cfbd4c6f51f8a883eb35a4288ed9c09
  * Your subject line is longer than 50 characters

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 19823405c6ba7908ca8ef0eada05fc160ff78d04
  * Your subject line is longer than 50 characters

 * Commit: 2073a9093297a7def14228a2ea899cb7b09ea97d
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 2b3b0baf3cfbd4c6f51f8a883eb35a4288ed9c09
  * Your subject line is longer than 50 characters

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

File: upload_to_codecov.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+# Exist when command returns not 0
+set -e
+
+# Ensure all Jenkins variables are set (e.g. commit, branch, etc.)
+if [ -e "jenkins-env" ]; then
+  source jenkins-env
+fi
+
+bash <(curl -s https://codecov.io/bash) -t 274c63b8-b698-425d-a0ab-6a4020eca599 -f coverage/coverage.json
```
Comment (maxandersen) [inline upload_to_codecov.sh]: what kind of access does this token have ?

**Comment by kwk** [comment]
[test]

File: upload_to_codecov.sh (hunk 1 above)
Comment (kwk) [inline upload_to_codecov.sh]: > what kind of access does this token have ?

@maxandersen on the settings page in codecov.io for the individual repository it says: "The token below is used exclusively for uploading coverage reports."

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 19823405c6ba7908ca8ef0eada05fc160ff78d04
  * Your subject line is longer than 50 characters

 * Commit: 2073a9093297a7def14228a2ea899cb7b09ea97d
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 2b3b0baf3cfbd4c6f51f8a883eb35a4288ed9c09
  * Your subject line is longer than 50 characters

 * Commit: b3621747d8933e8d2b54d17af3c29b7ba9c66acd
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by corinnekrych** [comment]
[test]

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 19823405c6ba7908ca8ef0eada05fc160ff78d04
  * Your subject line is longer than 50 characters

 * Commit: 2073a9093297a7def14228a2ea899cb7b09ea97d
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 2b3b0baf3cfbd4c6f51f8a883eb35a4288ed9c09
  * Your subject line is longer than 50 characters

 * Commit: b3621747d8933e8d2b54d17af3c29b7ba9c66acd
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 2d07c64b84a68c3b8f024b4971610a54aa1c814b
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 19823405c6ba7908ca8ef0eada05fc160ff78d04
  * Your subject line is longer than 50 characters

 * Commit: 2073a9093297a7def14228a2ea899cb7b09ea97d
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 2b3b0baf3cfbd4c6f51f8a883eb35a4288ed9c09
  * Your subject line is longer than 50 characters

 * Commit: b3621747d8933e8d2b54d17af3c29b7ba9c66acd
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 2d07c64b84a68c3b8f024b4971610a54aa1c814b
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 1667180a2b3c06c93eb8d4cddf5d23c32cd16a64
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Your subject line is longer than 50 characters

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by fabric8cd** [comment]
@corinnekrych snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1368-fabric8-ui.int.rdu2c.fabric8.io

**Comment by joshuawilson** [comment]
It looks like the build failed on the PR

**Comment by joshuawilson** [comment]
[test]

**Comment by fabric8cd** [comment]
@corinnekrych snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1368-fabric8-ui.int.rdu2c.fabric8.io

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 19823405c6ba7908ca8ef0eada05fc160ff78d04
  * Your subject line is longer than 50 characters

 * Commit: 2073a9093297a7def14228a2ea899cb7b09ea97d
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 2b3b0baf3cfbd4c6f51f8a883eb35a4288ed9c09
  * Your subject line is longer than 50 characters

 * Commit: b3621747d8933e8d2b54d17af3c29b7ba9c66acd
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 2d07c64b84a68c3b8f024b4971610a54aa1c814b
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 1667180a2b3c06c93eb8d4cddf5d23c32cd16a64
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Your subject line is longer than 50 characters

 * Commit: 78ee19bcad6055732491bbc30a51b672f6926abf
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by fabric8cd** [comment]
@corinnekrych snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1368-fabric8-ui.int.rdu2c.fabric8.io

**Comment by kwk** [comment]
[test]

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 19823405c6ba7908ca8ef0eada05fc160ff78d04
  * Your subject line is longer than 50 characters

 * Commit: 2073a9093297a7def14228a2ea899cb7b09ea97d
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 2b3b0baf3cfbd4c6f51f8a883eb35a4288ed9c09
  * Your subject line is longer than 50 characters

 * Commit: b3621747d8933e8d2b54d17af3c29b7ba9c66acd
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 2d07c64b84a68c3b8f024b4971610a54aa1c814b
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 1667180a2b3c06c93eb8d4cddf5d23c32cd16a64
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Your subject line is longer than 50 characters

 * Commit: 78ee19bcad6055732491bbc30a51b672f6926abf
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by fabric8cd** [comment]
@corinnekrych snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1368-fabric8-ui.int.rdu2c.fabric8.io

**Comment by kwk** [comment]
[test]

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 19823405c6ba7908ca8ef0eada05fc160ff78d04
  * Your subject line is longer than 50 characters

 * Commit: 2073a9093297a7def14228a2ea899cb7b09ea97d
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 2b3b0baf3cfbd4c6f51f8a883eb35a4288ed9c09
  * Your subject line is longer than 50 characters

 * Commit: b3621747d8933e8d2b54d17af3c29b7ba9c66acd
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 2d07c64b84a68c3b8f024b4971610a54aa1c814b
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 1667180a2b3c06c93eb8d4cddf5d23c32cd16a64
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Your subject line is longer than 50 characters

 * Commit: 78ee19bcad6055732491bbc30a51b672f6926abf
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 19823405c6ba7908ca8ef0eada05fc160ff78d04
  * Your subject line is longer than 50 characters

 * Commit: 2073a9093297a7def14228a2ea899cb7b09ea97d
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 2b3b0baf3cfbd4c6f51f8a883eb35a4288ed9c09
  * Your subject line is longer than 50 characters

 * Commit: b3621747d8933e8d2b54d17af3c29b7ba9c66acd
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 2d07c64b84a68c3b8f024b4971610a54aa1c814b
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 1667180a2b3c06c93eb8d4cddf5d23c32cd16a64
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Your subject line is longer than 50 characters

 * Commit: 78ee19bcad6055732491bbc30a51b672f6926abf
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 2726086a1de0016a0df0030a9bdfe66cafc300a0
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by fabric8cd** [comment]
@corinnekrych snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1368-fabric8-ui.int.rdu2c.fabric8.io

**Comment by fabric8cd** [comment]
@corinnekrych snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1368-fabric8-ui.int.rdu2c.fabric8.io

**Comment by kwk** [comment]
[test]

**Comment by kwk** [comment]
[test]

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 19823405c6ba7908ca8ef0eada05fc160ff78d04
  * Your subject line is longer than 50 characters

 * Commit: 2073a9093297a7def14228a2ea899cb7b09ea97d
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 2b3b0baf3cfbd4c6f51f8a883eb35a4288ed9c09
  * Your subject line is longer than 50 characters

 * Commit: b3621747d8933e8d2b54d17af3c29b7ba9c66acd
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 2d07c64b84a68c3b8f024b4971610a54aa1c814b
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 1667180a2b3c06c93eb8d4cddf5d23c32cd16a64
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Your subject line is longer than 50 characters

 * Commit: 78ee19bcad6055732491bbc30a51b672f6926abf
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 2726086a1de0016a0df0030a9bdfe66cafc300a0
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by fabric8cd** [comment]
@corinnekrych snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1368-fabric8-ui.int.rdu2c.fabric8.io

**Comment by joshuawilson** [comment]
[test]

**Comment by fabric8cd** [comment]
@corinnekrych snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1368-fabric8-ui.int.rdu2c.fabric8.io

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 19823405c6ba7908ca8ef0eada05fc160ff78d04
  * Your subject line is longer than 50 characters

 * Commit: 2073a9093297a7def14228a2ea899cb7b09ea97d
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 2b3b0baf3cfbd4c6f51f8a883eb35a4288ed9c09
  * Your subject line is longer than 50 characters

 * Commit: b3621747d8933e8d2b54d17af3c29b7ba9c66acd
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 2d07c64b84a68c3b8f024b4971610a54aa1c814b
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 1667180a2b3c06c93eb8d4cddf5d23c32cd16a64
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Your subject line is longer than 50 characters

 * Commit: 78ee19bcad6055732491bbc30a51b672f6926abf
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 2726086a1de0016a0df0030a9bdfe66cafc300a0
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by fabric8cd** [comment]
@corinnekrych snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1368-fabric8-ui.int.rdu2c.fabric8.io

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 19823405c6ba7908ca8ef0eada05fc160ff78d04
  * Your subject line is longer than 50 characters

 * Commit: 2073a9093297a7def14228a2ea899cb7b09ea97d
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 2b3b0baf3cfbd4c6f51f8a883eb35a4288ed9c09
  * Your subject line is longer than 50 characters

 * Commit: b3621747d8933e8d2b54d17af3c29b7ba9c66acd
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 2d07c64b84a68c3b8f024b4971610a54aa1c814b
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 1667180a2b3c06c93eb8d4cddf5d23c32cd16a64
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Your subject line is longer than 50 characters

 * Commit: 78ee19bcad6055732491bbc30a51b672f6926abf
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 2726086a1de0016a0df0030a9bdfe66cafc300a0
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by fabric8cd** [comment]
@corinnekrych snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1368-fabric8-ui.int.rdu2c.fabric8.io

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 19823405c6ba7908ca8ef0eada05fc160ff78d04
  * Your subject line is longer than 50 characters

 * Commit: 2073a9093297a7def14228a2ea899cb7b09ea97d
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 2b3b0baf3cfbd4c6f51f8a883eb35a4288ed9c09
  * Your subject line is longer than 50 characters

 * Commit: b3621747d8933e8d2b54d17af3c29b7ba9c66acd
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 2d07c64b84a68c3b8f024b4971610a54aa1c814b
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 1667180a2b3c06c93eb8d4cddf5d23c32cd16a64
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Your subject line is longer than 50 characters

 * Commit: 78ee19bcad6055732491bbc30a51b672f6926abf
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 2726086a1de0016a0df0030a9bdfe66cafc300a0
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 19823405c6ba7908ca8ef0eada05fc160ff78d04
  * Your subject line is longer than 50 characters

 * Commit: 2073a9093297a7def14228a2ea899cb7b09ea97d
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 2b3b0baf3cfbd4c6f51f8a883eb35a4288ed9c09
  * Your subject line is longer than 50 characters

 * Commit: b3621747d8933e8d2b54d17af3c29b7ba9c66acd
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 2d07c64b84a68c3b8f024b4971610a54aa1c814b
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 1667180a2b3c06c93eb8d4cddf5d23c32cd16a64
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Your subject line is longer than 50 characters

 * Commit: 78ee19bcad6055732491bbc30a51b672f6926abf
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 2726086a1de0016a0df0030a9bdfe66cafc300a0
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by kwk** [comment]
[test]

**Comment by kwk** [comment]
[test]

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 19823405c6ba7908ca8ef0eada05fc160ff78d04
  * Your subject line is longer than 50 characters

 * Commit: 2073a9093297a7def14228a2ea899cb7b09ea97d
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 2b3b0baf3cfbd4c6f51f8a883eb35a4288ed9c09
  * Your subject line is longer than 50 characters

 * Commit: b3621747d8933e8d2b54d17af3c29b7ba9c66acd
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 2d07c64b84a68c3b8f024b4971610a54aa1c814b
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 1667180a2b3c06c93eb8d4cddf5d23c32cd16a64
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Your subject line is longer than 50 characters

 * Commit: 78ee19bcad6055732491bbc30a51b672f6926abf
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 2726086a1de0016a0df0030a9bdfe66cafc300a0
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by corinnekrych** [comment]
Thanks @kwk for the hard work.
Coverage report available for this branch here: https://codecov.io/gh/fabric8io/fabric8-ui/branch/codecov.setup
@joshuawilson ok to merge?

**Pull request merged**